bot.user.js

```diff
@@ -436,17 +437,14 @@ var bot = (function() {
             if (window.autoRespawn && !window.playing && bot.isBotEnabled &&
                 bot.ranOnce && !bot.isBotRunning) {
                 bot.connectBot();
-                if (document.querySelector('div#lastscore').childNodes
-                    .length > 1) {
-                    window.scores.push(parseInt(document.querySelector(
-                        'div#lastscore').childNodes[1].innerHTML));
-                }
+                /* The lastscore should be checked here in the future,
+                but not only when bot, just when not playing.
+                THIS SHOULD ONLY BE EXECUTED ONCE! */
+                userInterface.saveScore(); // Checks highscore
```

Comment:
Should I just move the whole userInterface-module up?
Or is there something like function prototyping in C?